src/courses/beginner/10.md

```diff
@@ -185,7 +185,13 @@ In the above example, all controls from `my_nginx` profile will be executed, **e
 
 ### Selectively Including Controls
 
-If there are only a handful of controls that should be executed from an included profile, it’s not necessary to skip all the unneeded controls, or worse, copy/paste those controls bit-for-bit into your profile[^first]. Instead, use the `require_controls` command.
+If there are only a handful of controls that should be executed from an included profile, it’s not necessary to skip all the unneeded controls, or worse, copy/paste those controls bit-for-bit into your profile. 
+
+:::warning Keep profiles in sync!
+Copying and pasting controls from a profile, instead of creating an overlay, can lead to important updates not being reflected. Overlays keep the profile changes in sync as they pull the latest updates.
```

Comment:
```suggestion
Copying and pasting controls from a profile, instead of creating an overlay, can lead to important updates from the upstream profile not being reflected in the new one. Overlays keep the profile changes in sync as they pull the latest updates from upstream before running.
```